<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>773</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_z">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>White.jpg</normaloff>White.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>White.jpg</normaloff>White.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_c">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>White.jpg</normaloff>White.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_v">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>White.jpg</normaloff>White.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_b">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>White.jpg</normaloff>White.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>White.jpg</normaloff>White.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>White.jpg</normaloff>White.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_s">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Black.jpg</normaloff>Black.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_d">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Black.jpg</normaloff>Black.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_g">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Black.jpg</normaloff>Black.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_h">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Black.jpg</normaloff>Black.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_j">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>60</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Black.jpg</normaloff>Black.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>120</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Длительность</string>
    </property>
    <widget class="QRadioButton" name="radioButton_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1/2</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1/4</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1/8</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1/16</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_p">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Пауза</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="sbut">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="obut">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>490</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rbut">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>590</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Сбросить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testB">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>610</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Октава</string>
    </property>
    <widget class="QRadioButton" name="radioButton_83">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_84">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_85">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_86">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbut">
    <property name="geometry">
     <rect>
      <x>904</x>
      <y>382</y>
      <width>101</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
